scripts/deploy.js

```diff
@@ -1,7 +1,8 @@
 const chromeWebstoreUpload = require('chrome-webstore-upload');
+const path = require('path');
 const fs = require('fs');
 
-const ZIP_PATH = '../release/hypercrx.zip';
+const ZIP_PATH = path.join(__dirname, '..', '/release/hypercrx.zip');
```

Comment:
"The relative path to a filename can be used. Remember, however, that this path will be relative to process.cwd()."

https://stackoverflow.com/a/44601028/10369621